### PR TITLE
upgrades quick-upgrade and upgradeable-planets

### PR DESCRIPTION
**Pull request by gabeio**
also adjusts javascript utils to properly call planetCanUpgrade

### CODE REVIEW

**Comment by gabeio** [comment]
apparently quick-upgrade doesn't seem to be working, still trying to figure out why.

**Comment by gabeio** [comment]
I also am going to try changing just the coords to be a link so it's much easier when you have something upgrade-able to be able to find it.

**Comment by phated** [comment]
@gabeio what is your discord handle?

**Comment by gabeio** [comment]
@phated also `@gabeio` :D I like to keep things simple lol

**Comment by gabeio** [comment]
Screen Shot 2021-05-31 at 18 30 26

added a horizontal break, I actually didn't realize the bottom half was for _all planets_.

**Comment by gabeio** [comment]
Oh I didn't know you could click on the line for the planet upgrade, this is ready then. I might refactor upgrade-able planets in a separate pr.

**Comment by jacobrosenthal** [comment]
Testing now. You want to touch the date so it moves to the top of the list?

File: javascript/utils/utils.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export let canPlanetUpgrade = (planet) => {
   if (!planet) {
     return false;
   }
-  return df.entityStore.planetCanUpgrade(planet)
+  return df.entityStore.constructor.planetCanUpgrade(planet)
```
Comment (gabeio) [inline javascript/utils/utils.js:36]: this is the only thing I'm not 100% sure about because I saw a discussion in discord about this change...

**Comment by gabeio** [comment]
> Testing now. You want to touch the date so it moves to the top of the list?

Done.

**Comment by gabeio** [comment]
Screen Shot 2021-05-31 at 19 52 40
Screen Shot 2021-05-31 at 19 52 30

**Pull request merged**